### DISCUSSION TITLE
How to remove modal content padding from Action

### DISCUSSION THREAD

**Discussion by lepresk** [filamentphp/filament#7642] [Help]
Hi everyone, is there a way to remove the padding of modal content when it is displayed through an action? This could be helpful in certain cases, such as when I need to insert custom content into the modal, for example

![Capture](https://github.com/filamentphp/filament/assets/10780686/62224345-955d-4b5a-91e3-17b699fc8823)

**Comment by pboivin** [comment]
You can do this with custom CSS. You can target the modal through the page selector:
```php
.filament-resources-list-records-page .filament-modal-header {
    /* ... */
}
```
